datafusion/sqllogictest/test_files/repartition.slt

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I ran this test locally without the code changes in this PR and this test still passes

Reply:
Hm, I ran the test locally before and it failed. Let me re-verify it.

Reply:
Fixed it. I added a few lines before which broken it.